etcd_3_4: 3.4.28 -> 3.4.37

superherointj opened the pull request:
etcd_3_4: 3.4.28 -> 3.4.37

- Release: https://github.com/etcd-io/etcd/releases/tag/v3.4.37
- Changelog: https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.4.md

* etcd_3_4: remove offline from maintainers for inactivity

CC @offlinehacker

superherointj commented on pkgs/top-level/all-packages.nix:
```diff
@@ -10136,7 +10136,7 @@ with pkgs;
 
   etcd = etcd_3_5;
-  etcd_3_4 = callPackage ../servers/etcd/3.4.nix { };
+  etcd_3_4 = callPackage ../servers/etcd/3_4 { };
```
The underscore (`_`) is on purpose to standardize. 3.5 is changed in another PR.

superherointj commented on pkgs/servers/etcd/3_4/default.nix:
```diff
@@ -6,33 +6,41 @@
   };
 
-  vendorHash = "sha256-DbDIz/cbXqYHaGqNqP+wYpuiaFiZYElRXmQwBXnCbMk=";
+  proxyVendor = true;
```
```suggestion
  proxyVendor = true;
```
This was necessary.

superherointj committed:
etcd_3_4: remove offline from maintainers for inactivity

superherointj commented:
CC @dtomvan

dtomvan reviewed (approved):
Diff LGTM, seems to build on ofborg

dtomvan commented:
Would be nice if the nixos test gets a different maintainer too I think. We should probably factor it out into a package test that can be run for each version of etcd, and then the "default" version as a toplevel nixostest. Then we could do `nix-build -A etcd_3_4.passthru.tests`...

ulrikstrid commented on pkgs/servers/etcd/3_4/default.nix:
```diff
@@ -6,33 +6,41 @@
+    license = lib.licenses.asl20;
     homepage = "https://etcd.io/";
-    maintainers = with maintainers; [ offline ];
+    maintainers = with lib.maintainers; [ ];
```
Did we drop @offline on purpose?

ulrikstrid replied:
Nvm, this has been adressed

superherointj committed:
etcd_3_4: 3.4.28 -> 3.4.37

Release: https://github.com/etcd-io/etcd/releases/tag/v3.4.37

Changelog: https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.4.md

superherointj commented:
@ofborg eval

superherointj commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 430420`
Commit: `4201d698d30f58cd4051bfce11f17db5309bf7fc`

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>etcd_3_4</li>
  </ul>
</details>

dtomvan reviewed (approved):
Aight
